src/components/common/Problem.vue

```diff
@@ -29,34 +30,107 @@ export default defineComponent({
 		const _file: Ref<string> = ref(props.file);
 		const renderDiv = ref<HTMLElement>();
 
+		// modified from https://github.com/tserkov/vue-plugin-load-script
+		async function loadScript(src: string) {
+			console.log('loading scripts...', src);
+			// eslint-disable-line no-param-reassign
+			return new Promise<void>(function (resolve, reject) {
+				let shouldAppend = false;
+				let el: HTMLScriptElement | HTMLLinkElement | null;
+				create_el: if (/\.js(?:\??[0-9a-zA-Z=]*)$/.exec(src)) {
+					el = document.querySelector('script[src="' + src + '"]');
+					if (el && el.hasAttribute('data-loaded')) {
+						resolve();
+						console.log(`${src} seems to have already been loaded!`);
+						return;
+					} else if (el) {
+						break create_el;
+					}
+
+					el = document.createElement('script');
+					el.type = 'text/javascript';
+					el.async = true;
+					el.src = src;
+					shouldAppend = true;
+				} else if (/\.css(?:\??[0-9a-zA-Z=]*)$/.exec(src)) {
+					el = document.querySelector('link[href="' + src + '"]');
+					if (el && el.hasAttribute('data-loaded')) {
+						console.log(`${src} seems to have already been loaded!`);
+						resolve();
+						return;
+					} else if (el) {
+						break create_el;
+					}
+
+					el = document.createElement('link');
+					el.type = 'text/css';
+					el.rel = 'stylesheet';
+					el.href = src;
+					shouldAppend = true;
+				} else {
+					console.error(`Received invalid src: ${src}`);
+					reject();
+					return;
+				}
+
+				el.onerror = reject;
+				el.onabort = reject;
+				el.onload = function loadScriptHandler() {
+					console.log(`[onLoad] ${src} successfully loaded!`);
+					el?.setAttribute('data-loaded', 'true');
+					resolve();
+				};
```

Comment:
These events should be added with addEventListener instead of assigning to the on... attributes.  So, for example, change to `el.addEventListener('error', reject);`.